plugin.audio.rne/.gitignore

```diff
@@ -0,0 +1,5 @@
+*.pyc
```

Comment:
Please remove from submission, add to gitattributes

Reply:
OK, removed.